### DISCUSSION TITLE
Time of Event not correct with Events from Wordpress Events Manager

### DISCUSSION THREAD

**Discussion by derlucas** [bitfireAT/icsx5#149] [Q&A]
I have trouble using the App with Calendars from Events Manager Wordpress Plugin.
The Events are saved with Timezone "Berlin" at 19:00 and shown correctly on the Wordpress Site. In the ICS file the Entry is like this:

`DTSTART;TZID=Europe/Berlin:20230421T190000
DTEND;TZID=Europe/Berlin:20230421T235900`

But in my Android Calendar [(Simple Mobile Tools Calendar](https://github.com/SimpleMobileTools/Simple-Calendar) the Event is show at 01:00 in the night instead of 19:00. I tested the Google Calendar App too, and it shows the same false Time.

What would be the best way to find the root of the Issue?

**Comment by erebion** [comment]
I am subscribed to the same calendar htat he subscribes to and use it with Etar, same result.

**Comment by ArnyminerZ** [comment]
That line of code doesn't seem correct. Are you missing a line break, or is the Workpress plugin not generating the ICS correctly? We've had [issues like this](https://github.com/bitfireAT/icsx5/discussions/86) before. And they were caused by the server, not ICSx⁵. Please, take a look, or provide the generated ICS file so we can analyze it. If you don't want to share the data publicly you can contact us privately.

**Reply by erebion** [reply]
The ICS link in question: https://www.chaostreff-dortmund.de/events.ics

**Reply by erebion** [reply]
Do you see an issue with this file?

**Reply by rfc2822** [reply]
Sorry, a bit late… but no, I don't see a problem. ICSx⁵ imports it well and the events are shown in the calendar app. Are there specific problems now?

**Reply by erebion** [reply]
Well, every entry is shown to be at 01:00.

**Comment by tikismoke** [comment]
I got the same issue with grocy ical file for meal plan.
All events are shown at 1am instead of the correctt hour in android default calendar:



`
DTSTART;TZID=Europe/Paris:20230908T120000
DTEND;TZID=Europe/Paris:20230908T120000
`

**Comment by rfc2822** [comment]
I could reproduce the problem and we'll have a look.

Please follow up at https://github.com/bitfireAT/ical4android/issues/112

**Comment by odin568** [comment]
Ok, same issue I also face here https://github.com/bitfireAT/icsx5/discussions/198
